### PR TITLE
[KSQL-12110] | Vedarth Sharma | Check if directory exists before cleanup 7.4.x

### PR DESCRIPTION
**Pull request by VedarthConfluent**
Fix 7.4.x ITs

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -716,7 +718,7 @@
                         <log4j.configuration>file:${project.basedir}/../ksqldb-test-util/src/main/resources/log4j.properties</log4j.configuration>
                     </systemPropertyVariables>
                     <argLine>-verbose:gc -Xloggc:"${project.build.directory}/gc.log"</argLine>
-                    <argLine>-Duser.timezone=UTC -Xmx1g -Xms1g -XX:MaxPermSize=256m</argLine>
+                    <argLine>-Duser.timezone=UTC -Xmx1g -Xms1g</argLine>
```
Comment (kagarwal06) [inline pom.xml:721]: Why is this change needed?

Comment (VedarthConfluent) [reply pom.xml:721]: It was breaking otherwise. We were using java 11 before now it is `. set-cp-java-version` tool

Comment (VedarthConfluent) [reply pom.xml:721]: https://semaphore.ci.confluent.io/jobs/16f55e26-0406-4169-9370-aa66fc303b6b

Comment (kagarwal06) [reply pom.xml:721]: So in the later version( i see the job is using java 17), this option is not supported

**Review by kagarwal06 — APPROVED** [review]

File: pom.xml (hunk 1 above)
Comment (VedarthConfluent) [reply pom.xml:721]: Yes, that's correct
